### PR TITLE
Add of possibility to remove email authentication with SSO.

### PR DESCRIPTION
**Pull request by Giggiux**
Didn't tested, but should work (adapted code from nodebb-plugin-sso-facebook)

### CODE REVIEW

File: library.js [hunk 1]
```diff
@@ -21,7 +21,9 @@
 		}
 	});
 
-	var Twitter = {};
```
Comment (julianlam) [inline library.js]: Please fix the indentation issues in this PR, as we use tabs, not spaces. Thanks.

**Comment by Giggiux** [comment]
I hope I did it right :)

File: templates/admin/plugins/sso-twitter.tpl [hunk 2]
```diff
@@ -26,10 +26,16 @@
 				<label for="secret">API Secret</label>
 				<input type="text" name="secret" id="secret" title="API Secret" class="form-control" placeholder="API Secret">
 			</div>
+			<div class="checkbox">
+						<label for="showSiteTitle" class="mdl-switch mdl-js-switch mdl-js-ripple-effect">
```
Comment (julianlam) [inline templates/admin/plugins/sso-twitter.tpl]: Here too 😄

**Comment by Giggiux** [comment]
O my... sometimes add spaces other times tab... gonna fix again :joy:

**Comment by julianlam** [comment]
Merged in via 9d179c670af93e9069308380df44d872e6819402

**Pull request closed without merging**